packages/components/src/combobox-control/types.ts

```diff
@@ -82,4 +82,8 @@ export type ComboboxControlProps = Pick<
 	 * The current value of the control.
 	 */
 	value?: string | null;
+	/**
+	 * The placeholder for the Combobox.
```

Comment:
Nit: Clarity for what we're referring to here ("combobox" as the element with combobox role vs. "ComboboxControl" as the component).

```suggestion
	 * The placeholder for the combobox.
```